common/src/thrift/file_master.thrift

```diff
@@ -43,47 +47,48 @@ service FileSystemMasterService {
 
   // Tachyon Client API
 
-  /*
+  /**
```

Comment:
Could you revert the javadoc-ing in this file too? I've opened https://tachyon.atlassian.net/browse/TACHYON-1267